app/assets/javascripts/admin/modules/deselect_taxon_click.js

```diff
@@ -0,0 +1,23 @@
+(function (Modules) {
+    "use strict";
+
+    Modules.DeselectTaxonClick = function () {
+        var helper = window.GOVUK.taxonomyTreeHelper;
+        this.start = function (container) {
+            var deselectTaxonClick = function () {
+                var taxon_name = $(container).closest('.taxon-breadcrumb').find('li:last').text() ;
+                $("input[data-taxon-name=\"" + taxon_name + "\"]").each(function(idx, el){
+                    $(el).prop('checked', false)
+                    if (!helper.hasCheckedSiblings(el)){
+                        helper.uncheckAncestors(el);
+                    }
+                });
+                $('.topic-tree').trigger('change');
+                return false;
```

Comment:
Not sure this is needed?

Reply:
In the process of fixing a performance issue, I've removed this whole file in 58850d095d4f7b5fa5a6cf64496b806185399cb5